app/types/transport/requests/task/change-task-status.ts

```diff
@@ -0,0 +1,30 @@
+import { NewMessage } from 'ctproto';
+
+export interface ChangeTaskStatusPayload {
+  /**
+   * Id of the task status of which should be updated
+   */
+  taskId: string
+
+  /**
+   * New task status if exists
+   */
+  newStatusId?: string
+
+  /**
+   * Id of status task previously had
```

Comment:
pls describe why we're sending it? Maybe the current status could be taken from by the `taskId`